Makefile

```diff
@@ -206,7 +206,9 @@ tags :
 bootstrap-json-%: phony
 	cabal v2-build --project=cabal.project.release --with-compiler=ghc-$* --dry-run cabal-install:exe:cabal
 	cp dist-newstyle/cache/plan.json bootstrap/linux-$*.plan.json
-	cabal v2-run -vnormal+stderr --builddir=dist-newstyle-bootstrap --project=cabal.project.bootstrap cabal-bootstrap-gen -- bootstrap/linux-$*.plan.json  | python3 -m json.tool | tee bootstrap/linux-$*.json
```

Comment:
i wonder why `-vnormal+stderr` was chosen in the first place

Reply:
I think it's because *if* the executable has already been built, it works (since it causes the immediate cabal-install output to go to stderr).